tool/grind.dart

```diff
@@ -0,0 +1,24 @@
+library polymer_interop.grind;
+
+import 'package:grinder/grinder.dart';
+
+const dartium = 'dartium';
+const chrome = 'chrome';
+const firefox = 'firefox';
+const safari = 'safari';
+
+const macPlatforms = const ['dartium,firefox,chrome,safari'];
```

Comment:
it might eventually be nice to allow command line args for this, but I think its fine to be hardcoded for now